### DISCUSSION TITLE
feat(datepicker): New option to not show the month name from the days table

### DISCUSSION THREAD

**Discussion by robmv** [angular/components#25171] [Ideas]
#### Feature Description

Add a new option to not show the month name on the days table and a way to set that as default for all calendars. Currently the current month is displayed two time, on the month-year selection control, and on the table. It looks redundant and the material.io components show the Android example that doesn't show that month, for comparison:

![android](https://user-images.githubusercontent.com/1652983/131576539-8a00ecef-056f-4cbd-bd7b-39fe5fd0f60e.png) ![angular](https://user-images.githubusercontent.com/1652983/131576347-1e06eb02-bbfc-4838-a22b-b4c64f2d735a.png)

#### Use Case

Avoid the duplicate month on the datepicker and save space

Note: This was noticed while implementing an adapter for js-joda, that probably will try to contribute if there is interest. Another thing I noticed is that on the adapter the getMonthNames() is only invoked with the 'short' parameter. It is planned for the future, or it is some legacy implementation detail?.

[a comment by an automated account is not shown]

**Comment by zarend** [comment]
Hello, this is not on our roadmap, and we don't have any plans to implement hiding the month name from the days table.

This is still open for discussion, and with a compeling reason or enough interest, this could become prioritized. I'm converting this issue to a discussion.
